src/logger.ts

```diff
@@ -0,0 +1,20 @@
+import pino from 'pino';
+import type { Logger } from 'pino';
+
+const level = process.env.REACT_APP_LOG_LEVEL
+  ?? process.env.NODE_ENV === 'production' ? 'error' : 'debug';
+
+const logger = pino({
+  name: 'data-viewer',
+  level,
+  browser: {
+    asObject: true,
+  },
+  transport: {
+    target: 'pino-pretty',
```

Comment:
On the server side, we would do `node ... | npx pino-pretty` but I suppose that's not available in the browser, therefore it makes sense to configure this transport straightaway, yes? I cannot quickly tell if this is what the pino folks recommend.

Reply:
Oops, excellent catch! I did not mean to commit this. :flushed: 

Unfortunately, `pino-pretty` does not work in the browser. I'll remove this - thanks!

Reply:
Fixed!